source/kraken/realtime.cpp

```diff
@@ -257,6 +256,53 @@ static nt::disruption::Effect get_trip_effect(nt::disruption::StopTimeUpdate::St
 }
 
 
+static nt::disruption::Effect get_trip_effect(transit_realtime::Alert_Effect effect) {
```

Comment:
Just by curiosity, what's the difference between a `transit_realtime::Alert_Effect` and a `nt::disruption::Effect` ? seems to be a perfect 1:1 mapping.

Reply:
The first is the Extension added in kirin.proto where as other is an enum class in kraken. Kraken transforms gtfs-rt effect to disruption.Effect during treatment and retransforms to disruption.severity.effect during response generation in pb_converter.

Reply:
It looks like we can just skip that conversion, and go directly to wordings when needed, no?
So maybe remove that enum `nt::disruption::Effect` if not used anywhere else.

So just change the output type of `get_calculated_trip_effect` to match it.
And change the input type of `get_wordings`